pyproject.toml

```diff
@@ -155,30 +155,21 @@ dependencies = [
     "apache-airflow-providers-cncf-kubernetes>=5.1.1",
     "apache-airflow-providers-docker>=3.5.0",
 ]
-post-install-commands = [
-  "pip uninstall -y apache-airflow-providers-common-io"
+# Airflow install with constraint file, Airflow versions < 2.7 require a workaround for PyYAML
+pre-install-commands = ["""
+    AIRFLOW_MINOR_VERSION=$(echo "{matrix:airflow}" | cut -c-3)
+    AIRFLOW_LESS_THAN_2_7=$(echo "$AIRFLOW_MINOR_VERSION < 2.7" | bc)
```

Comment:
This method to compare versions may not be accurate:
```
$ echo "2.1 < 2.7" | bc    
1
$  echo "2.10 < 2.7" | bc
1
```

Reply:
Good catch! Updated the check of the minor version.